aim/config.py

```diff
@@ -84,8 +72,12 @@
     cfg.BoolOpt('recovery_restart', default=True,
                 help=("Set to True if you want the agents to exit in critical "
                       "situations.")),
+    cfg.StrOpt('aim_service_identifier', default=socket.gethostname(),
+               help="(NODB) Identifier for this specific AID service, "
```

Comment:
I think it is relevant for operators that need to know whether a restart is required or not while changing an option. I haven't opted for something more readable because we will have to copy-paste this string for every non persistent option.